src/components/element-editor/basic-details/basic-details.styles.tsx

```diff
@@ -1,8 +1,35 @@
+import { Text } from "@lifesg/react-design-system";
+import { Accordion } from "@lifesg/react-design-system/accordion";
 import styled from "styled-components";
 
 export const MandatoryFieldBox = styled.div`
     background: rgba(248, 248, 248, 1);
     padding: 1rem;
     display: flex;
     flex-direction: column;
+    margin-bottom: 2rem;
+`;
+
+export const FieldEditorAccordionItem = styled(Accordion.Item)`
+    #content-container {
+        padding: 0;
+    }
+
+    [data-testid="accordion-item-expandable-container"] {
+        padding: 0 1rem;
+        display: flex;
+        flex-direction: column;
+    }
+
+    > *:not(:last-child) {
+        margin-bottom: 1rem;
+    }
```

Comment:
what is the intention of this style? there is only 1 direct descendant so it is never applied

Reply:
ok I understand what you are trying to achieve now, instead of achieve the 2rem bottom spacing through the last child, you can apply 2rem bottom padding to `FieldEditorAccordionItem`. this way will be clearer because you can achieve the 2rem spacing through a single element